src/bin.js

```diff
@@ -104,6 +103,26 @@ commander
   })
   .action(request.numberBuy.bind(request));
 
+commander
+  .command('numbers:buy [number_pattern]')
+  .description('Buy a number to use for voice or SMS. If a --country_code is provided then the number_pattern is used to search for a number in the given country.')
+  .option('-c, --country_code [country_code]', 'the country code')
+  .option('--confirm', 'skip confirmation step and directly buy the number' )
+  .on('--help', () => {
+    emitter.log('  Examples:');
+    emitter.log(' ');
+    emitter.log('    $ nexmo number:buy 445555555555');
```

Comment:
Could you please match the emitter logs to the command name? they say `number:buy` but they should say `numbers:buy` for this command.

Reply:
woops! what an embarrassing oversight!